scratch/app-zoom.jsonc

```diff
@@ -0,0 +1,73 @@
+{
+  "type": "view",
+  "bindings": [
+    {
+      "type": "view",
+      "bindings": [
+        {
+          "type": "markdown",
+          "label": "##### Your Personal Meeting ID\n440 287 4306\nhttps://mattermost.zoom.us/my/matt.birtch"
+        },
+        {
+          "type": "button",
+          "label": "Start a Meeting",
+          "icon": "icons/zoom-icon-small.png",
```

Comment:
The blue color on the screenshot: how much is Zoom style and how much is mattermost style? (sorry for my "color blindness" but the two blues seem pretty similar :P )
Do we want to allow any level of customization in terms of colors here?

Reply:
It's all mattermost style. The only thing using the zoom blue is the app icon itself. Everything else uses standard mattermost theme colors